apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/pac/FissClaimTransformerV2.java

```diff
@@ -431,6 +449,28 @@ private List<Claim.ItemComponent> getClaimItems(RdaFissClaim claimGroup) {
               itemComponent.addModifier(createModifierCoding(revenueLine.getHcpcModifier3(), "3"));
               itemComponent.addModifier(createModifierCoding(revenueLine.getHcpcModifier4(), "4"));
 
+              if (Strings.isNotBlank(revenueLine.getNdcQty())
+                  && Strings.isNotBlank(revenueLine.getNdcQtyQual())) {
+                // Both rev_units_billed and ndc_qty can exist in the same sequence. Since
+                // rev_serv_unit_cnt serves
+                // the same purpose as rev_units_billed and since rev_serv_unit_cnt is set as an
+                // extension, ndc_qty
+                // can be as the quantity thus being fine with overwriting rev_units_billed
+                Quantity quantity = new Quantity();
+                try {
+                  quantity.setValue(Double.parseDouble(revenueLine.getNdcQty()));
+                } catch (NumberFormatException ex) {
+                  // If the NDC_QTY isn't a valid number, do not set quantity value. Awaiting
+                  // upstream RDA test data changes
+                  log.error("captured exception: message={}", ex.getMessage(), ex);
```

Comment:
A more specific error message might be helpful - "failed to parse ndcQty as a number" or something like that